### PR TITLE
fix: fixes bug when an annotation path in an http rule does not have =.

### PR DESCRIPTION
**Pull request by atulep**
Fixes a bug when an annotation path in an http rule does not have "=". The generator would not include such annotations into its metadata before sending a gRPC call.

For example, 

`/v1b3/projects/{project_id}/locations/{location}/jobs` would not be included in the metadata.

### CODE REVIEW

File: gapic/schema/wrappers.py [hunk 1]
```diff
@@ -1200,7 +1200,8 @@ def field_headers(self) -> Sequence[FieldHeader]:
 
         http = self.options.Extensions[annotations_pb2.http]
 
-        pattern = re.compile(r'\{([a-z][\w\d_.]+)=')
+        # Copied from Node generator.
```
Comment (davidcavazos) [inline gapic/schema/wrappers.py:1203]: Thanks! Can we add a link to the GitHub file where this regex was copied from?

**Review by davidcavazos — APPROVED** [review]

**Comment by parthea** [comment]
Aza, Please could you take a look at the test failure?

File: gapic/schema/wrappers.py (hunk 1 above)
Comment (atulep) [reply gapic/schema/wrappers.py:1207]: Done

**Review by vam-google — APPROVED** [review]
LGTM

**Pull request merged**